version.gradle

```diff
@@ -1,10 +1,10 @@
 allprojects {
 	ext {
-		shortVersion = '1.1.0-alpha2'
```

Comment:
Please don't bump the version. I'll do so one I release 1.1.0. Which I will do probably very soon :)